locales/pl.yml

```
@@ -101,6 +101,7 @@ changed-password-suc: Hasło zostało zmienione pomyślnie
 channel: Kanał
 channel-create: Utwórz kanał
 channel-deleted: Kanał usunięty
+channel-forced-devices: Urządzenia z wymuszonym kanałem
```

I don't know pl but maybe would be better do the same?
```suggestion
channel-forced-devices: Urządzenia z Wymuszonym Wanałem
```